salt/modules/cmdmod.py

```diff
@@ -416,16 +421,44 @@ def _get_stripped(cmd):
             raise CommandExecutionError(
                 'User \'{0}\' is not available'.format(runas)
             )
+
+    if group:
+        if salt.utils.platform.is_windows():
+            msg = 'group is not currently available on Windows'
+            raise SaltInvocationError(msg)
+        if not find_executable('sudo'):
```

Comment:
As with the comment on the import above, let's make sure this is a private function (i.e. `_find_executable`).

Reply:
Good catch! But I'll try to switch this over to `which_bin` since that's already imported.